_pages/dev/lazy-loading-guide.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Lazy Loading Guide
+---
+
+Lazy loading, also known as code splitting, lets you divide your JavaScript code into multiple chunks. The result is that you do not have to load all of the chunks when a user first accesses your storefront. Instead, chunks are only loaded when they are needed.
```

Comment:
> The result is that you do not have to load all of the chunks when a user first accesses your storefront. Instead, chunks are only loaded when they are needed.

The result is that you do not have to load all the JavaScript of the full application when a user accesses the first page. Instead, only the chunks required for the given page are loaded. While navigating the storefront, additional chunks might be loaded when needed.

Reply:
Beautiful! 😄